spec/lib/wiki_course_output_spec.rb

```diff
@@ -44,6 +44,22 @@
                       timeline_start: '2016-01-11',
                       timeline_end: '2016-04-24',
                       weeks: [week1, week2])
+      campaign1 = create(:campaign,
+                         id: 1,
```

Comment:
These IDs are not necessary. If you leave them out, IDs will be generated automatically.

Reply:
Ok...is the rest ok?

Reply:
tes, it looks reasonable.

Reply:
@ragesoss I have removed the ids in the latest commit...

Reply:
@ragesoss can you suggest me a rails issue i can try to solve next?

Reply:
@ragesoss can you please tell me why this failing in travis?
`rspec ./spec/features/admin_role_spec.rb:144 # Admin users linking a course to its Salesforce record makes the Link to Salesforce button appear`

Reply:
I haven't made any changes around this....

Reply:
That's a test that occasionally fails randomly. I've restarted the build.

Reply:
@ragesoss i am facing another issue which is unrelated..
`rspec ./spec/features/multiwiki_assignment_spec.rb:69 # multiwiki assignments will create a valid assignment for multilingual wikisource projects`